applications/serial_lte_modem/doc/GNSS_AT_commands.rst

```diff
@@ -118,6 +116,9 @@ Syntax
 Response syntax
 ~~~~~~~~~~~~~~~
 
+..
+   gps_read_response_start
+
 ::
 
    #XGPS: <gnss_service>,<gnss_status>
```

Comment:
The includes are failing without an error in the build process, and the end result is just adding this to the build:
![image](https://github.com/nrfconnect/sdk-nrf/assets/83582261/bda84c24-a061-40d7-a153-f50354af195a)

There's two ways to fix this:
```suggestion
.. gps_read_response_start

.. code-block::

   #XGPS: <gnss_service>,<gnss_status>
```
or
```suggestion

::

   #XGPS: <gnss_service>,<gnss_status>
   

.. gps_read_response_start
```

Reply:
One more hitch, it seems. The content to be included needs to appear in the file before the command to include it. Otherwise the command is, basically, finding itself and causing some of instances of the artifacting.